### DISCUSSION TITLE
Just a quick suggestion for you guys to attract more developers

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#152] [Developer Feedback]
The thing is that your app development method is quiet old fashioned in today's time. Now like in wear os app development they do drag and drop in the simulator which cuts a lot of time and effort. And for adding some complicated or logics or click functions we could add them by coding . 

Another example to make you understand is like making a website . In platforms people just drag the text which they wanted to write and place where ever they want just like in simulator I want you to put similar things like drag and drop. 

This method will save our time and help developers to put less effort and make apps faster then before which will significantly grow your app ecosystem.
And it will attract new developers as they will consider easier.

**Comment by MyoungXUE** [comment]
Hello, thank you very much for your advice!

Are you referring to a visual development tool, we currently have a similar tool for our watchface production https://docs.zepp.com/docs/guides/tools/watchface/

But for Mini Program, implementing such a system would require a lot of resources, and we will consider the idea at the right time.
